airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -110,6 +112,12 @@ def __init__(self, wasb_conn_id: str = default_conn_name, public_read: bool = Fa
         self.public_read = public_read
         self.connection = self.get_conn()
 
+        logger = logging.getLogger("azure.core.pipeline.policies.http_logging_policy")
+        try:
+            logger.setLevel(os.environ.get("AZURE_HTTP_LOGGING_LEVEL", logging.WARNING))
```

Comment:
Why?

If this configurability is really desired, I’d use `airflow.settings` for this.

```python
from airflow import settings

logger.setLevel(getattr(settings, "AZURE_HTTP_LOGGING_LEVEL", logging.WARNING))
```

So a user can add `AZURE_HTTP_LOGGING_LEVEL` to `airflow_local_settings` if they so desire. This will need an entry in documentation as well.

With that, I won’t add the `ValueError` block. If the user provides something that crashes Airflow, that’s the user’s problem and we don’t need to do things too magically.

Reply:
I can make those updates and reopen in a new pull request since the bot closed this one. I was mostly thinking the user might want to tweak the log level for these in particular and I was unaware of the standard settings approach. Thanks for the feedback!

Reply:
I can re-open if you wish to continue work on it (you will need to rebase to latest main)